### PR TITLE
Feature/Main HTML Markup

### PR DESCRIPTION
**Pull request by miraclmoore**
I have started on 2 of the 3 sections of the Main HTML Markup for the site. 

I have separated the site into 3 Sections to make it easier for myself:
Section 1: Blog or Blog Content
Section 2: What We Do or Services
Section 3: Testimonials

There are still some things I need to address:
1. Are we using the blog text, links, pics, etc...from the site? 
(Currently Section 1 just has placeholders, blank info, and Lorem Ipsum Text)
2. I just coded the body without integrating what Willa and Camper have coded. Is that not the process?
3. I'm not so good with figuring out the columns and bootstrap so that still needs to be fixed for Section 1
4. I need to figure out how to create the widget at the end of Section 1 as well.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Comment Deleted

**Review by ghost — CHANGES_REQUESTED** [review]
This is great work so far! I like how you divided the main section into three subsections. This really shows that you're getting better with Git and GitHub!

Before we can accept this pull request, you need to remove @willatauro's commits since those were done on the `master` branch accidentally. I think you can do this by pulling from `staging` and then manually deleting the `Dummy1` file if needed. If you can't remove the commits, don't worry about it and just delete the `Dummy1` file and push to your fork.

Also, after you pull from `staging` into your feature branch (`branch/feature/main-html-markup`), copy and paste your changes from `main.html` into `index.html` and then delete `main.html`.

When you've tested everything to make sure the page looks (acts) the way you want it to, you can push your changes to your forked repo's `branch/feature/main-html-markup` branch and the pull request should update automatically.

I'll be notified by GitHub when this happens (hopefully) and will review your code further at that time.

Does that make sense? Let me know if you have any problems with any of this. Thanks for doing this great work!

File: index.html [hunk 1]
```diff
@@ -32,6 +29,139 @@ <h1 class="company-name">Eleva<span>tion</span></h1>
...
+  </div>
+  </a>
+</section>
+</body>
+
+  
+
         <!-- BANNER, BUILT BY -->
         <div class="banner">
         </div>
```
Comment (ghost) [inline index.html]: Lines 165-167 are part of the `header` element at the beginning of the body and need to be moved up there. If we merge this code the way it is now, leaving this code here, it'll overwrite the correct code currently in the `staging` branch, essentially creating a bug.

File: index.html [hunk 2]
```diff
@@ -32,6 +29,139 @@ <h1 class="company-name">Eleva<span>tion</span></h1>
           <a href="#">Menu <span class="fa fa-bars"></span></a>
         </div>
```
Comment (ghost) [inline index.html:32]: The `header` element needs a closing `</header>` tag. If we merge this code the way it is now, without the closing header tag, it'll overwrite the correct code currently in the `staging` branch, essentially creating a bug.

File: index.html [hunk 3]
```diff
@@ -32,6 +29,139 @@ <h1 class="company-name">Eleva<span>tion</span></h1>
...
 
+
+<!--BODY, BUILT BY CMOORE-->
+<body>
```
Comment (ghost) [inline index.html]: The opening `<body>` tag needs to be below the closing `</head>` tag and above the `<!--HEADER, BUILT BY CAMPER-->` comment on line 14. If we merge this code the way it is now, with the opening body tag where it is, it'll overwrite the correct code currently in the staging branch, essentially creating a bug.

File: index.html [hunk 4]
```diff
@@ -32,6 +29,139 @@ <h1 class="company-name">Eleva<span>tion</span></h1>
...
+
+<!--SECTION ONE BLOG CONTENT--> 
+<section class="a" id="blog">
```
Comment (ghost) [inline index.html]: What does the `a` class mean? Can you make this more descriptive so we know what it's for? If it's a placeholder, it's better to remove the `class="a"` code entirely.

**Review by ghost — CHANGES_REQUESTED** [review]
So far so good! :+1: It's all in the `index.html` file now. If you can, delete the `Dummy1` file. If not, it's not a big deal as we'll be able to delete it after this pull request is accepted.

There are a few things that need to be changed before accepting this. Change those (should be quick changes), I'll check the code one last time, and that'll probably be enough to get this merged into `staging`. :1st_place_medal:

**Comment by miraclmoore** [comment]
Hey @camper-fcc! I'll make these changes and submit a new PR. I couldn't figure out how to delete the "Dummy1" file though.

**Review by ghost — APPROVED** [review]

**Review by shalinidc — APPROVED** [review]
Looks good to me 👍🏻

**Pull request merged**